AxonDeepSeg/integrity_test.py

```diff
@@ -32,7 +32,7 @@ def integrity_test():
 
         # Launch the axon and myelin segmentation on test image sample provided in the installation
         print('Computing the segmentation of axon and myelin on test image.')
-        axon_segmentation(path_testing, [str(path_testing / image)], path_model, acquired_resolution=0.13, overlap_value=[48,48])
+        axon_segmentation(path_testing, [str(path_testing / image)], path_model, acquired_resolution=0.13, gpu_id=0, overlap_value=[48,48])
```

Comment:
Note to the reviewer:
The CI test are not running on GPU, so I did not add a specific test for the GPU ID feature.
However, I added the GPU ID 0 in the integrity test so when we run the tests manually on a GPU cluster, the new feature is taken into account.

Reply:
Noted =)